packages/odf/components/kms-config/utils.tsx

```diff
@@ -645,6 +750,21 @@ export const kmsConfigValidation = (
         kmsObj.clientKey.value !== ''
       );
     }
+    case ProviderNames.AZURE: {
+      const kmsObj = kms as AzureConfig;
+      return (
+        kmsObj.name.valid &&
+        kmsObj.azureVaultURL.valid &&
+        kmsObj.clientID.valid &&
+        kmsObj.tenantID.valid &&
+        !kmsObj.clientCert.error &&
+        kmsObj.name.value !== '' &&
+        kmsObj.azureVaultURL.value !== '' &&
+        kmsObj.clientID.value !== '' &&
+        kmsObj.tenantID.value !== '' &&
+        kmsObj.clientCert.value !== ''
```

Comment:
```suggestion
        kmsObj.name.valid &&
        kmsObj.azureVaultURL.valid &&
        kmsObj.clientID.valid &&
        kmsObj.tenantID.valid &&
        !kmsObj.clientCert.error &&
```
is this empty check is required? The ```valid``` property also has a check for empty value. 
Here, https://github.com/red-hat-storage/odf-console/pull/1229/files#diff-d4e15f6d1df3eaf7bcabade00418e3aed07ee8e54cf81962275b74eabfc546c2R49

you are already checking empty,  azureStateClone.name.valid = name !== '' && isValidName(name);